main.go

```diff
@@ -258,3 +313,19 @@ func main() {
 		glog.Info("Received SIGTERM, exiting gracefully...")
 	}
 }
+
+// Returns intiliazed config, allows local usage (outside cluster) based on host kubeconfig or in-cluter
+func initKubeConfig() *rest.Config {
+
+	kubeConfigLocation := filepath.Join(os.Getenv("HOME"), ".kube", "config")
```

Comment:
this location is very hardcoded, do you mind to add a dedicated `kubeconfig` location? If unset we could fallback to the in-cluster-config.